demisto_sdk/commands/common/docker_images_metadata.py

```diff
@@ -0,0 +1,61 @@
+from typing import Dict, Optional
+
+from packaging.version import Version
+from pydantic import BaseModel
+
+from demisto_sdk.commands.common.constants import DOCKERFILES_INFO_REPO
+from demisto_sdk.commands.common.git_content_config import GitContentConfig
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.common.singleton import Singleton
+from demisto_sdk.commands.common.tools import get_remote_file_from_api
+
+DOCKER_IMAGES_METADATA_NAME = "docker_images_metadata.json"
+
+
+class DockerImageTagMetadata(BaseModel):
+    python_version: Optional[str]
+
+
+class DockerImagesMetadata(Singleton, BaseModel):
+    docker_images: Dict[str, Dict[str, DockerImageTagMetadata]]
+
+    @classmethod
+    def from_github(
+        cls, file_name: str = DOCKER_IMAGES_METADATA_NAME, tag: str = "master"
+    ):
+        tag = "4c162e56174bec3ee7bb1b418ad2b20e4bdce3e0"
```

Comment:
what's this hardcoded value?

Reply:
the commit of the dockerfiles-info PR, it will be the master once https://github.com/demisto/dockerfiles-info/pull/6 will be merged, at the meantime its used for testing